--- cpputest-3.7.x-a/cmake/Scripts/CppUTestBuildTimeDiscoverTests.cmake	2016-05-25 07:41:47.000000000 +0200
+++ cpputest-3.7.x-b/cmake/Scripts/CppUTestBuildTimeDiscoverTests.cmake	2020-05-08 16:42:30.757033332 +0200
@@ -56,28 +56,6 @@ set(CTESTFNAME "${CMAKE_CURRENT_BINARY_D
 file(STRINGS ${CTESTFNAME} CTESTTESTS)
 set(EXECUTABLE_CMD "${CMAKE_CURRENT_BINARY_DIR}/${EXECUTABLE}")
 
-if (TESTS_DETAILED)
-  set(DISCOVER_ARG "-ln")
-  buildtime_discover_tests("${EXECUTABLE_CMD}" "${DISCOVER_ARG}" TestList_GroupsAndNames)
-  set(lastgroup "")
-  foreach(testfullname ${TestList_GroupsAndNames})
-    string(REGEX MATCH "^([^/.]+)" groupname ${testfullname})
-    string(REGEX MATCH "([^/.]+)$" testname ${testfullname})
-    if (NOT ("${groupname}" STREQUAL "${lastgroup}"))
-      message("TestGroup: ${groupname}:")
-      set(lastgroup "${groupname}")
-    endif (NOT ("${groupname}" STREQUAL "${lastgroup}"))
-    message("... ${testname}")
-    buildtime_add_test(${EXECUTABLE}.${testfullname} ${EXECUTABLE_CMD} -sg ${groupname} -sn ${testname} -c)
-  endforeach()
-else (TESTS_DETAILED)
-  set(DISCOVER_ARG "-lg")
-  buildtime_discover_tests("${EXECUTABLE_CMD}" "${DISCOVER_ARG}" TestList_Groups)
-  foreach(group ${TestList_Groups})
-    message("TestGroup: ${group}")
-    buildtime_add_test(${EXECUTABLE}.${group} "${EXECUTABLE_CMD}" -sg ${group} -c)
-  endforeach()
-endif (TESTS_DETAILED)
 
 
 # create separate CTest test for each CppUTestTests test
--- cpputest-3.7.x-a/include/CppUTest/CppUTestConfig.h	2016-05-25 07:41:47.000000000 +0200
+++ cpputest-3.7.x-b/include/CppUTest/CppUTestConfig.h	2020-05-08 16:41:19.589283920 +0200
@@ -172,14 +172,6 @@
  * Works on non-Visual C++ compilers and Visual C++ 2008 and newer
  */
 
-#if CPPUTEST_USE_STD_C_LIB && (!defined(_MSC_VER) || (_MSC_VER >= 1800))
-#define CPPUTEST_HAVE_FENV
-#if defined(__WATCOMC__)
-#define CPPUTEST_FENV_IS_WORKING_PROPERLY 0
-#else
-#define CPPUTEST_FENV_IS_WORKING_PROPERLY 1
-#endif
-#endif
 
 /*
  * Detection of different 64 bit environments
