diff -uNrp r1/src/library/example/basic_receiver_to_pulseaudio.c r2/src/library/example/basic_receiver_to_pulseaudio.c
--- r1/src/library/example/basic_receiver_to_pulseaudio.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/library/example/basic_receiver_to_pulseaudio.c	2020-05-17 22:28:26.350619678 +0200
@@ -30,7 +30,7 @@
 #define MY_RECEIVER_REPAIR_PORT 10102
 
 /* Signal parameters. */
-#define MY_SAMPLE_RATE 44100
+#define MY_SAMPLE_RATE 48000
 #define MY_NUM_CHANNELS 2
 #define MY_BUFFER_SIZE 1000
 
diff -uNrp r1/src/library/example/basic_sender_from_pulseaudio.c r2/src/library/example/basic_sender_from_pulseaudio.c
--- r1/src/library/example/basic_sender_from_pulseaudio.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/library/example/basic_sender_from_pulseaudio.c	2020-05-17 22:28:26.350619678 +0200
@@ -31,7 +31,7 @@
 #define MY_RECEIVER_REPAIR_PORT 10102
 
 /* Signal parameters */
-#define MY_SAMPLE_RATE 44100
+#define MY_SAMPLE_RATE 48000
 #define MY_NUM_CHANNELS 2
 #define MY_BUFFER_SIZE 1000
 
diff -uNrp r1/src/library/example/basic_sender_sine_wave.c r2/src/library/example/basic_sender_sine_wave.c
--- r1/src/library/example/basic_sender_sine_wave.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/library/example/basic_sender_sine_wave.c	2020-05-17 22:28:26.350619678 +0200
@@ -29,7 +29,7 @@
 #define MY_RECEIVER_REPAIR_PORT 10102
 
 /* Signal parameters */
-#define MY_SAMPLE_RATE 44100
+#define MY_SAMPLE_RATE 48000
 #define MY_SINE_RATE 440
 #define MY_SINE_DURATION (MY_SAMPLE_RATE * 10)
 #define MY_BUFFER_SIZE 100
diff -uNrp r1/src/library/example/send_receive_rtp.c r2/src/library/example/send_receive_rtp.c
--- r1/src/library/example/send_receive_rtp.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/library/example/send_receive_rtp.c	2020-05-17 22:28:26.350619678 +0200
@@ -31,7 +31,7 @@
 #define MY_RECEIVER_REPAIR_PORT 10202
 
 /* Signal parameters */
-#define MY_SAMPLE_RATE 44100
+#define MY_SAMPLE_RATE 48000
 #define MY_BUFFER_SIZE 2000
 
 #define oops()                                                                           \
diff -uNrp r1/src/library/example/send_receive_rtp_with_fecframe.c r2/src/library/example/send_receive_rtp_with_fecframe.c
--- r1/src/library/example/send_receive_rtp_with_fecframe.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/library/example/send_receive_rtp_with_fecframe.c	2020-05-17 22:28:26.350619678 +0200
@@ -34,7 +34,7 @@
 #define MY_RECEIVER_REPAIR_PORT 10202
 
 /* Signal parameters */
-#define MY_SAMPLE_RATE 44100
+#define MY_SAMPLE_RATE 48000
 #define MY_BUFFER_SIZE 2000
 
 #define oops()                                                                           \
diff -uNrp r1/src/library/src/config_helpers.cpp r2/src/library/src/config_helpers.cpp
--- r1/src/library/src/config_helpers.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/library/src/config_helpers.cpp	2020-05-17 22:28:26.350619678 +0200
@@ -44,10 +44,10 @@ bool sender_config_from_user(pipeline::S
         return false;
     }
 
-    if (in.packet_sample_rate != 0 && in.packet_sample_rate != 44100) {
+    if (in.packet_sample_rate != 0 && in.packet_sample_rate != 48000) {
         roc_log(LogError,
                 "bad configuration:"
-                " invalid packet_sample_rate, only 44100 is currently supported");
+                " invalid packet_sample_rate, only 48000 is currently supported");
         return false;
     }
 
diff -uNrp r1/src/modules/roc_audio/depacketizer.cpp r2/src/modules/roc_audio/depacketizer.cpp
--- r1/src/modules/roc_audio/depacketizer.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/modules/roc_audio/depacketizer.cpp	2020-05-17 22:28:26.350619678 +0200
@@ -24,7 +24,7 @@ inline void write_zeros(sample_t* buf, s
 
 inline void write_beep(sample_t* buf, size_t bufsz) {
     for (size_t n = 0; n < bufsz; n++) {
-        buf[n] = (sample_t)std::sin(2 * M_PI / 44100 * 880 * n);
+        buf[n] = (sample_t)std::sin(2 * M_PI / 48000 * 880 * n);
     }
 }
 
diff -uNrp r1/src/modules/roc_pipeline/config.h r2/src/modules/roc_pipeline/config.h
--- r1/src/modules/roc_pipeline/config.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/modules/roc_pipeline/config.h	2020-05-17 22:28:26.350619678 +0200
@@ -29,7 +29,7 @@ namespace roc {
 namespace pipeline {
 
 //! Default sample rate, number of samples per second.
-const size_t DefaultSampleRate = 44100;
+const size_t DefaultSampleRate = 48000;
 
 //! Default channel mask.
 const packet::channel_mask_t DefaultChannelMask = 0x3;
diff -uNrp r1/src/modules/roc_rtp/format_map.cpp r2/src/modules/roc_rtp/format_map.cpp
--- r1/src/modules/roc_rtp/format_map.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/modules/roc_rtp/format_map.cpp	2020-05-17 22:28:26.350619678 +0200
@@ -33,7 +33,7 @@ FormatMap::FormatMap()
         Format fmt;
         fmt.payload_type = PayloadType_L16_Mono;
         fmt.flags = packet::Packet::FlagAudio;
-        fmt.sample_rate = 44100;
+        fmt.sample_rate = 48000;
         fmt.channel_mask = 0x1;
         fmt.get_num_samples = audio::PCM_int16_1ch.samples_from_payload_size;
         fmt.new_encoder =
@@ -46,7 +46,7 @@ FormatMap::FormatMap()
         Format fmt;
         fmt.payload_type = PayloadType_L16_Stereo;
         fmt.flags = packet::Packet::FlagAudio;
-        fmt.sample_rate = 44100;
+        fmt.sample_rate = 48000;
         fmt.channel_mask = 0x3;
         fmt.get_num_samples = audio::PCM_int16_2ch.samples_from_payload_size;
         fmt.new_encoder =
diff -uNrp r1/src/modules/roc_rtp/headers.h r2/src/modules/roc_rtp/headers.h
--- r1/src/modules/roc_rtp/headers.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/modules/roc_rtp/headers.h	2020-05-17 22:28:26.354619665 +0200
@@ -27,8 +27,8 @@ enum Version {
 
 //! RTP payload type.
 enum PayloadType {
-    PayloadType_L16_Stereo = 10, //!< Audio, 16-bit samples, 2 channels, 44100 Hz.
-    PayloadType_L16_Mono = 11    //!< Audio, 16-bit samples, 1 channel, 44100 Hz.
+    PayloadType_L16_Stereo = 10, //!< Audio, 16-bit samples, 2 channels, 48000 Hz.
+    PayloadType_L16_Mono = 11    //!< Audio, 16-bit samples, 1 channel, 48000 Hz.
 };
 
 //! RTP header.
diff -uNrp r1/src/pulse/module-roc-sink.c r2/src/pulse/module-roc-sink.c
--- r1/src/pulse/module-roc-sink.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/pulse/module-roc-sink.c	2020-05-17 22:28:26.354619665 +0200
@@ -220,7 +220,7 @@ int pa__init(pa_module* m) {
     /* prepare sample spec and channel map used in this sink */
     pa_sample_spec sample_spec;
     sample_spec.format = PA_SAMPLE_FLOAT32LE;
-    sample_spec.rate = 44100;
+    sample_spec.rate = 48000;
     sample_spec.channels = 2;
 
     pa_channel_map channel_map;
@@ -267,7 +267,7 @@ int pa__init(pa_module* m) {
     roc_sender_config sender_config;
     memset(&sender_config, 0, sizeof(sender_config));
 
-    sender_config.frame_sample_rate = 44100;
+    sender_config.frame_sample_rate = 48000;
     sender_config.frame_channels = ROC_CHANNEL_SET_STEREO;
     sender_config.frame_encoding = ROC_FRAME_ENCODING_PCM_FLOAT;
 
diff -uNrp r1/src/pulse/module-roc-sink-input.c r2/src/pulse/module-roc-sink-input.c
--- r1/src/pulse/module-roc-sink-input.c	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/pulse/module-roc-sink-input.c	2020-05-17 22:28:26.354619665 +0200
@@ -159,7 +159,7 @@ int pa__init(pa_module* m) {
     /* prepare sample spec and channel map used in this sink */
     pa_sample_spec sample_spec;
     sample_spec.format = PA_SAMPLE_FLOAT32LE;
-    sample_spec.rate = 44100;
+    sample_spec.rate = 48000;
     sample_spec.channels = 2;
 
     pa_channel_map channel_map;
@@ -213,7 +213,7 @@ int pa__init(pa_module* m) {
     roc_receiver_config receiver_config;
     memset(&receiver_config, 0, sizeof(receiver_config));
 
-    receiver_config.frame_sample_rate = 44100;
+    receiver_config.frame_sample_rate = 48000;
     receiver_config.frame_channels = ROC_CHANNEL_SET_STEREO;
     receiver_config.frame_encoding = ROC_FRAME_ENCODING_PCM_FLOAT;
 
diff -uNrp r1/src/tests/roc_audio/test_mixer.cpp r2/src/tests/roc_audio/test_mixer.cpp
--- r1/src/tests/roc_audio/test_mixer.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_audio/test_mixer.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -20,7 +20,7 @@ namespace audio {
 
 namespace {
 
-enum { BufSz = 100, SampleRate = 44100,
+enum { BufSz = 100, SampleRate = 48000,
        ChannelMask = 0x1, MaxBufSz = 500 };
 
 const core::nanoseconds_t MaxBufDuration =
diff -uNrp r1/src/tests/roc_library/test_context.cpp r2/src/tests/roc_library/test_context.cpp
--- r1/src/tests/roc_library/test_context.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_library/test_context.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -54,7 +54,7 @@ TEST(context, reference_counting) {
     {
         roc_sender_config sender_config;
         memset(&sender_config, 0, sizeof(sender_config));
-        sender_config.frame_sample_rate = 44100;
+        sender_config.frame_sample_rate = 48000;
         sender_config.frame_channels = ROC_CHANNEL_SET_STEREO;
         sender_config.frame_encoding = ROC_FRAME_ENCODING_PCM_FLOAT;
 
@@ -67,7 +67,7 @@ TEST(context, reference_counting) {
         {
             roc_receiver_config receiver_config;
             memset(&receiver_config, 0, sizeof(receiver_config));
-            receiver_config.frame_sample_rate = 44100;
+            receiver_config.frame_sample_rate = 48000;
             receiver_config.frame_channels = ROC_CHANNEL_SET_STEREO;
             receiver_config.frame_encoding = ROC_FRAME_ENCODING_PCM_FLOAT;
 
diff -uNrp r1/src/tests/roc_library/test_helpers/utils.h r2/src/tests/roc_library/test_helpers/utils.h
--- r1/src/tests/roc_library/test_helpers/utils.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_library/test_helpers/utils.h	2020-05-17 22:28:26.354619665 +0200
@@ -18,7 +18,7 @@ namespace {
 enum {
     MaxBufSize = 500,
 
-    SampleRate = 44100,
+    SampleRate = 48000,
     NumChans = 2,
 
     SourcePackets = 10,
diff -uNrp r1/src/tests/roc_library/test_receiver.cpp r2/src/tests/roc_library/test_receiver.cpp
--- r1/src/tests/roc_library/test_receiver.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_library/test_receiver.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -27,7 +27,7 @@ TEST_GROUP(receiver) {
         CHECK(context);
 
         memset(&receiver_config, 0, sizeof(receiver_config));
-        receiver_config.frame_sample_rate = 44100;
+        receiver_config.frame_sample_rate = 48000;
         receiver_config.frame_channels = ROC_CHANNEL_SET_STEREO;
         receiver_config.frame_encoding = ROC_FRAME_ENCODING_PCM_FLOAT;
     }
diff -uNrp r1/src/tests/roc_library/test_sender.cpp r2/src/tests/roc_library/test_sender.cpp
--- r1/src/tests/roc_library/test_sender.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_library/test_sender.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -28,7 +28,7 @@ TEST_GROUP(sender) {
 
         memset(&sender_config, 0, sizeof(sender_config));
         sender_config.fec_code = ROC_FEC_DISABLE;
-        sender_config.frame_sample_rate = 44100;
+        sender_config.frame_sample_rate = 48000;
         sender_config.frame_channels = ROC_CHANNEL_SET_STEREO;
         sender_config.frame_encoding = ROC_FRAME_ENCODING_PCM_FLOAT;
     }
diff -uNrp r1/src/tests/roc_pipeline/test_converter_sink.cpp r2/src/tests/roc_pipeline/test_converter_sink.cpp
--- r1/src/tests/roc_pipeline/test_converter_sink.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_pipeline/test_converter_sink.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -23,7 +23,7 @@ namespace {
 enum {
     MaxBufSize = 1000,
 
-    SampleRate = 44100,
+    SampleRate = 48000,
     ChMask = 0x3,
     NumCh = 2,
 
diff -uNrp r1/src/tests/roc_pipeline/test_converter_source.cpp r2/src/tests/roc_pipeline/test_converter_source.cpp
--- r1/src/tests/roc_pipeline/test_converter_source.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_pipeline/test_converter_source.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -23,7 +23,7 @@ namespace {
 enum {
     MaxBufSize = 1000,
 
-    SampleRate = 44100,
+    SampleRate = 48000,
     ChMask = 0x3,
     NumCh = 2,
 
diff -uNrp r1/src/tests/roc_pipeline/test_receiver_source.cpp r2/src/tests/roc_pipeline/test_receiver_source.cpp
--- r1/src/tests/roc_pipeline/test_receiver_source.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_pipeline/test_receiver_source.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -30,7 +30,7 @@ const rtp::PayloadType PayloadType = rtp
 enum {
     MaxBufSize = 500,
 
-    SampleRate = 44100,
+    SampleRate = 48000,
     ChMask = 0x3,
     NumCh = 2,
 
diff -uNrp r1/src/tests/roc_pipeline/test_sender_sink.cpp r2/src/tests/roc_pipeline/test_sender_sink.cpp
--- r1/src/tests/roc_pipeline/test_sender_sink.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_pipeline/test_sender_sink.cpp	2020-05-17 22:28:26.354619665 +0200
@@ -31,7 +31,7 @@ rtp::PayloadType PayloadType = rtp::Payl
 enum {
     MaxBufSize = 1000,
 
-    SampleRate = 44100,
+    SampleRate = 48000,
     ChMask = 0x3,
     NumCh = 2,
 
diff -uNrp r1/src/tests/roc_pipeline/test_sender_sink_receiver_source.cpp r2/src/tests/roc_pipeline/test_sender_sink_receiver_source.cpp
--- r1/src/tests/roc_pipeline/test_sender_sink_receiver_source.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_pipeline/test_sender_sink_receiver_source.cpp	2020-05-17 22:28:26.358619652 +0200
@@ -29,7 +29,7 @@ namespace {
 enum {
     MaxBufSize = 500,
 
-    SampleRate = 44100,
+    SampleRate = 48000,
     ChMask = 0x3,
     NumCh = 2,
 
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.h r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.h
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.h	2020-05-17 22:28:26.358619652 +0200
@@ -48,7 +48,7 @@ static PacketInfo rtp_l16_1ch_10s_12ext
   /* num_channels     */ 1u,
   /* num_samples      */ 10u,
   /* samplebits       */ 16u,
-  /* samplerate       */ 44100u,
+  /* samplerate       */ 48000u,
 
   /* samples          */ {
     /* channel #0       */ { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 },
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.json r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.json
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.json	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_12ext.json	2020-05-17 22:28:26.358619652 +0200
@@ -20,7 +20,7 @@
     "num_channels": 1,
     "num_samples": 10,
     "samplebits": 16,
-    "samplerate": 44100,
+    "samplerate": 48000,
     "samples": [
         [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
     ]
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.h r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.h
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.h	2020-05-17 22:28:26.358619652 +0200
@@ -49,7 +49,7 @@ static PacketInfo rtp_l16_1ch_10s_4pad_2
   /* num_channels     */ 1u,
   /* num_samples      */ 10u,
   /* samplebits       */ 16u,
-  /* samplerate       */ 44100u,
+  /* samplerate       */ 48000u,
 
   /* samples          */ {
     /* channel #0       */ { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 },
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.json r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.json
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.json	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_1ch_10s_4pad_2csrc_12ext_marker.json	2020-05-17 22:28:26.358619652 +0200
@@ -20,7 +20,7 @@
     "num_channels": 1,
     "num_samples": 10,
     "samplebits": 16,
-    "samplerate": 44100,
+    "samplerate": 48000,
     "samples": [
         [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
     ]
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.h r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.h
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.h	2020-05-17 22:28:26.358619652 +0200
@@ -152,7 +152,7 @@ static PacketInfo rtp_l16_2ch_300s_80pad
   /* num_channels     */ 2u,
   /* num_samples      */ 300u,
   /* samplebits       */ 16u,
-  /* samplerate       */ 44100u,
+  /* samplerate       */ 48000u,
 
   /* samples          */ {
     /* channel #0       */ {
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.json r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.json
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.json	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_300s_80pad.json	2020-05-17 22:28:26.358619652 +0200
@@ -20,7 +20,7 @@
     "num_channels": 2,
     "num_samples": 300,
     "samplebits": 16,
-    "samplerate": 44100,
+    "samplerate": 48000,
     "samples": [
         [
             -22, -23, -24, -25, -26, -26, -27, -27, -27, -27, -28,
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.h r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.h
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.h	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.h	2020-05-17 22:28:26.358619652 +0200
@@ -152,7 +152,7 @@ static PacketInfo rtp_l16_2ch_320s = {
   /* num_channels     */ 2u,
   /* num_samples      */ 320u,
   /* samplebits       */ 16u,
-  /* samplerate       */ 44100u,
+  /* samplerate       */ 48000u,
 
   /* samples          */ {
     /* channel #0       */ {
diff -uNrp r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.json r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.json
--- r1/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.json	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_rtp/test_packets/rtp_l16_2ch_320s.json	2020-05-17 22:28:26.358619652 +0200
@@ -20,7 +20,7 @@
     "num_channels": 2,
     "num_samples": 320,
     "samplebits": 16,
-    "samplerate": 44100,
+    "samplerate": 48000,
     "samples": [
         [
             -22, -23, -24, -25, -26, -26, -27, -27, -27, -27, -28,
diff -uNrp r1/src/tests/roc_sndio/target_sox/test_pump.cpp r2/src/tests/roc_sndio/target_sox/test_pump.cpp
--- r1/src/tests/roc_sndio/target_sox/test_pump.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_sndio/target_sox/test_pump.cpp	2020-05-17 22:28:26.358619652 +0200
@@ -24,7 +24,7 @@ namespace sndio {
 
 namespace {
 
-enum { BufSize = 512, SampleRate = 44100, ChMask = 0x3 };
+enum { BufSize = 512, SampleRate = 48000, ChMask = 0x3 };
 
 const core::nanoseconds_t BufDuration =
     BufSize * core::Second / (SampleRate * packet::num_channels(ChMask));
diff -uNrp r1/src/tests/roc_sndio/target_sox/test_sox_sink.cpp r2/src/tests/roc_sndio/target_sox/test_sox_sink.cpp
--- r1/src/tests/roc_sndio/target_sox/test_sox_sink.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_sndio/target_sox/test_sox_sink.cpp	2020-05-17 22:28:26.358619652 +0200
@@ -17,7 +17,7 @@ namespace sndio {
 
 namespace {
 
-enum { FrameSize = 500, SampleRate = 44100, ChMask = 0x3 };
+enum { FrameSize = 500, SampleRate = 48000, ChMask = 0x3 };
 
 const core::nanoseconds_t FrameDuration =
     FrameSize * core::Second / (SampleRate * packet::num_channels(ChMask));
diff -uNrp r1/src/tests/roc_sndio/target_sox/test_sox_source.cpp r2/src/tests/roc_sndio/target_sox/test_sox_source.cpp
--- r1/src/tests/roc_sndio/target_sox/test_sox_source.cpp	2020-05-10 14:59:53.000000000 +0200
+++ r2/src/tests/roc_sndio/target_sox/test_sox_source.cpp	2020-05-17 22:28:26.358619652 +0200
@@ -26,7 +26,7 @@ namespace {
 enum {
     MaxBufSize = 8192,
     FrameSize = 500,
-    SampleRate = 44100,
+    SampleRate = 48000,
     ChMask = 0x3,
     NumChans = 2
 };
